app/models/education_benefits_claim.rb

```diff
@@ -3,7 +3,9 @@
 require 'attr_encrypted'
 class EducationBenefitsClaim < ApplicationRecord
   FORM_TYPES = %w[1990 1995 1990e 5490 5495 1990n 0993 0994 1995s].freeze
-
+  FORM_HEADERS = [
+    '22-1990', '22-1995', '22-1990e', '22-5490', '22-5495', '22-1990n', '22-0993', '22-0994', '22-1995 STEM'
```

Comment:
Definitely makes labeling clearer.

Not asking for it, but this is getting pretty close to a real data structure where there's an ordered set of form types and form headers. These two arrays are next to each other so its pretty quick to debug if they get mismatched but there's also the option to do something like

```
FORMS=
[
  {code: '1990', header: '22-1990'},
...
  {code: '1995s', header: '22-1995 STEM'},
]
```

That would let you be even more specific and clear in the loops

`FORM_TYPE_HEADERS = EducationBenefitsClaim::FORMS.map { |form|       [form[:header], '', ''] }` 

or something similar. 

But that means rewriting everything that already uses `FORM_TYPES`

In any case now the headers are separate from the types which is a plus 👍